stock_putaway_product_form/views/product.xml

```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl-3). -->
+<odoo>
+    <record id="view_template_form_putaway" model="ir.ui.view">
```

Comment:
Maybe.... ¿Can we remove this views? In v13.0 exists a smartbutton to view in other list view the rules...

Reply:
I'm not sure about this, can you tell me what is the view on the smartbutton you mention? (how to access the view).

Reply:
Ok.. :smile:

Reply:
Sorry @victoralmau ... Here is some photos...

![image](https://user-images.githubusercontent.com/7689807/100121247-33ce2300-2e79-11eb-8360-0280e81c1d6c.png)

![image](https://user-images.githubusercontent.com/7689807/100121315-3a5c9a80-2e79-11eb-8b28-eade0020eafe.png)

Reply:
Thanks a lot, removed now.

Reply:
@sergio-teruel @victoralmau The aim of this module... was to put putaways in product form directly... so its name. @sebastienbeau @kevinkhao